lib/core/form/components/form-renderer.component.scss

```
@@ -106,6 +106,34 @@
             font-weight: bold;
         }
     }
+
+    &-compact-field-container {
+        display: flex;
+        font-size: 12px;
```

shouldn't we use font size relative to theme variables?
```suggestion
        font-size: var(--theme-caption-font-size);
```

---

Not need anymore